### PR TITLE
Componentize container directive, add tests for config.

### PR DESCRIPTION
**Pull request by jmuharsky**

### CODE REVIEW

File: client/components/container/container-config-directive.html [hunk 1]
```diff
@@ -1,19 +1,20 @@
-<div class="sidebar-group">
-  <div class="sidebar-group-title">Container</div>
+<div>
 
-  <div class="sidebar-item">
-    <div class="sidebar-item-label">display columns</div>
-    <div class="sidebar-item-value">
+  <div class="pk-sidebar-item">
+    <div class="pk-sidebar-item-label">display columns</div>
+    <div class="pk-sidebar-item-value">
```
Comment (cmccoy) [inline client/components/container/container-config-directive.html]: Should this change be in #121?

File: client/components/container/container-directive.js [hunk 2]
```diff
@@ -36,42 +36,14 @@ goog.provide('p3rf.perfkit.explorer.components.container.ContainerDirective');
  * @return {Object} Directive definition object.
  */
 p3rf.perfkit.explorer.components.container.
-ContainerDirective = function() {
+ContainerDirective = function(dashboardService) {
```
Comment (cmccoy) [inline client/components/container/container-directive.js]: dashboardService doesn't appear to be used.

File: client/components/container/container-config-directive.html (hunk 1 above)
Comment (jmuharsky) [inline client/components/container/container-config-directive.html]: Yes.  Reverted to sidebar-

File: client/components/container/container-directive.js (hunk 2 above)
Comment (jmuharsky) [inline client/components/container/container-directive.js]: Removed.

**Comment by jmuharsky** [comment]
Addressed comments.  PTAL?

File: client/components/container/container-config-directive_test.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
+/**
+ * @copyright Copyright 2014 Google Inc. All rights reserved.
```
Comment (cmccoy) [inline client/components/container/container-config-directive_test.js]: 2015?

**Comment by cmccoy** [comment]
LGTM.

File: client/components/container/container-config-directive_test.js (hunk 3 above)
Comment (jmuharsky) [inline client/components/container/container-config-directive_test.js]: Done.

**Pull request merged**